pkgs/tools/misc/btdu/default.nix

```diff
@@ -1,23 +1,23 @@
 {stdenv, lib, fetchurl, dub, ncurses, ldc, zlib, removeReferencesTo }:
 
 let
-    _d_ae_ver              = "0.0.3100";
+    _d_ae_ver              = "0.0.3141";
```

Comment:
Can you explain to me why is this changed and the others are not?

Reply:
Thanks for the review! There is no version change for those inputs, so there nothing to change them to:

https://github.com/dlang-community/containers/releases
https://github.com/D-Programming-Deimos/ncurses/tags